pkg/api/mlflow/dao/repositories/helpers.go

```diff
@@ -30,3 +33,22 @@ func makeParamConflictPlaceholdersAndValues(params []models.Param) (string, []in
 	}
 	return placeholders, valuesArray
 }
+
+// AddJsonCondition adds where condition(s) to the query to select items having the specified map of json paths
+// and values in the given json column. Json path is expressed as "key" or "outerkey.nestedKey".
+func AddJsonCondition(tx *gorm.DB, jsonColumnName string, jsonPathValueMap map[string]string) {
```

Comment:
Testme.

Reply:
tested in integration `get_histories_test.go`, which validates the sql with dbs.

Reply:
Thank you!